aiida/orm/implementation/django/querybuilder.py

```diff
@@ -206,11 +206,11 @@ def get_filter_expr(self, operator, value, attr_key, is_attribute, alias=None, c
                     ''.format(operator, value)
                 )
         elif operator == 'in':
+            if not value:
```

Comment:
what if `value` is not even an interable? This will raise a `TypeError`, so maybe it is better to first check that a list/tuple is passed.

Reply:
Change implemented. In the end I ended up using a try/except statement instead of checking for explicit types (`value` could also be a set maybe?) because apparently "checking for iterables" is kind of controversial and this is a more "pythonic" approach ([sauce](https://stackoverflow.com/a/1952481/638366)). Let me know if you still prefer explicit type checking.

Reply:
Normally that depends on your interface whether it declares a specific type, but in this case I don't think we do so I like your solution :+1: